release-pipeline.yml

```diff
@@ -2166,8 +2166,32 @@ stages:
               displayName: 'Getting Python script'
               workingDirectory: $(root_dir)
 
-            - script: |
-                pip install azure-storage-blob
-                python3 releaseVersionUpdate.py "$(VERSION_CNT_SAS_URL)" "`azcopy --version`"
-              displayName: 'Updating version number'
-              workingDirectory: $(root_dir)
+            - task: AzureCLI@2
+              displayName: 'Updating version number with Azure CLI'
+              inputs:
+                azureSubscription: 'ESRP KeyVault identity'
+                scriptType: bash
+                scriptLocation: inlineScript
+                inlineScript: |
+                  pip install azure-storage-blob
+                  AZCOPY_AUTO_LOGIN_TYPE=AzCLI
+                  executable_name=azcopy 
+                  
+                  #Extract the version number 
+                  azcopy_version=$(azcopy --version | awk '{print $3}' | head -n 1)
+                  echo "azcopy version: $azcopy_version"
+                  
+                  #Upload to release version blob file
+                  azcopy --version | awk '{print $3}' | head -n 1 | azcopy copy https://azcopyvnextrelease.blob.core.windows.net/releasemetadata/latest_version.txt \
```

Comment:
can you first try this script in your VM? might make it easier to catch the error. 

I think the env var Auto login type isnt being recognized

Reply:
I tried it yesterday with no errors. I'm making some changes and try it again in my VM